src/administration/views.py

```diff
@@ -39,3 +40,15 @@ class ArticleViewSet(ReadOnlyModelViewSet):
 class NewsletterViewSet(ReadOnlyModelViewSet):
     serializer_class = NewsletterSerializer
     queryset = Newsletter.objects.filter(is_published=True)
+
+
+class WantToHelpViewSet(generics.ListCreateAPIView):
```

Comment:
there will be only one WantToHelp added at a time. ListCreateAPIView it's not necessary. Use APIView/ViewSet that allows adding only one entity